app/controllers/patients_controller.rb

```diff
@@ -93,7 +98,31 @@ def find_patient
     @patient = Patient.find params[:id]
   end
 
+  def format_dates(params)
```

Comment:
This is for formatting things at save, right? I think we might be able to ditch it if we swap the date fields from `text_field` to `date_field`. We should def keep this kinda thing for the audit trail deal though. Let's try it and see what happens.